cherab/solps/formats/raw_simulation_files.py

```diff
@@ -30,7 +30,7 @@
 
 
 # Code based on script by Felix Reimold (2016)
-def load_solps_from_raw_output(simulation_path, debug=False):
+def load_solps_from_raw_output(simulation_path, baserun_path=None, debug=False):
```

Comment:
This will break existing code which passes `debug` as a positional argument. Please put `baserun_path` as  the third argument instead.

Reply:
Done.